test/functional/plugins_test.rb

```diff
@@ -71,6 +72,29 @@
 #=========================================================================================#
 #                           CliCommand plugin type
 #=========================================================================================#
+describe 'cli command plugins' do
+  include FunctionalHelper
+
+  it 'is able to respond to a plugin-based cli subcommand' do
+    outcome = inspec_with_env('meaningoflife answer',  INSPEC_CONFIG_DIR: File.join(config_dir_path, 'meaning_by_path'))
```

Comment:
`meaning_by_path` and other phrases brought to you by Enlightenment Through Programming.